mypy/stubgenc.py

```diff
@@ -92,7 +92,9 @@ def add_typing_import(output: List[str]) -> List[str]:
 
 
 def is_c_function(obj: object) -> bool:
-    return inspect.isbuiltin(obj) or type(obj) is type(ord)
+    return inspect.isbuiltin(obj) or \
+           type(obj) is type(ord) or \
+           type(obj).__name__ == 'cython_function_or_method'
```

Comment:
I don't know if there's any particular reason. If `inspect` works, we should perhaps use it? 

As long as Cython doesn't enforce the use of optional types, it would at least be useful to be able to detect which functions or modules were compiled by Cython, and generate wider types (as I described above) for them.